app/routes/agenda/agendaitems/agendaitem/index.js

```diff
@@ -31,6 +31,8 @@ export default class DetailAgendaitemAgendaitemsAgendaRoute extends Route {
     }
     const agendaItemTreatment = await model.treatment;
     this.newsletterInfo = await agendaItemTreatment?.newsletterInfo;
+    this.decisionActivity = await agendaItemTreatment?.decisionActivity;
+    await this.decisionActivity?.decisionResultCode;
```

Comment:
Is it necessary to load the data here?  It will automatically be fetched by using `this.model.treatment.decisionActivity.decisionResultCode` in the template?

Reply:
Only having it in the template would work, but that would be an "await in the template" even though we don't use `await`.
In later stories I am using the tracked `decisionActivity`.